public/app/plugins/datasource/grafana-azure-monitor-datasource/partials/query.editor.html

```diff
@@ -10,7 +10,7 @@
     <div class="gf-form" ng-if="ctrl.target.queryType === 'Azure Monitor' || ctrl.target.queryType === 'Azure Log Analytics'">
       <label class="gf-form-label query-keyword width-9">Subscription</label>
       <gf-form-dropdown model="ctrl.target.subscription" allow-custom="true" lookup-text="true"
-        get-options="ctrl.subscriptions" on-change="ctrl.onSubscriptionChange()" css-class="min-width-12">
+        get-options="ctrl.getSubscriptions($query)" on-change="ctrl.onSubscriptionChange()" css-class="min-width-12">
```

Comment:
Doesn't like like `getSubscriptions` takes any arguments.

Reply:
Yeah, not using it. The form dropdown sends the $query value but I'm not using it for filtering so can remove it. One second.